.cursor/commands/commit-push-pr.en.md

```
@@ -0,0 +1,197 @@
+git commit -m "$MSG" && \
+git push -u origin "$BRANCH"
+
+# At this point, ask AI to create a PR, for example:
+# - Infer intent from the branch name
+# - Inspect changes via: git diff --name-status
+# - Generate PR title and body
+# - Create the PR via mcp_github_create_pull_request or `gh pr create`
+```
+
+> Note: MCP (Model Context Protocol) is a standard protocol for safely allowing agents to interact with external services like GitHub.  
+> To use it for PR creation, you will need an MCP-compatible environment and GitHub integration. See: https://modelcontextprotocol.io/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)